test/functional/api/v2/template_controller_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_plugin_helper'
+
+class Api::V2::TemplateControllerTest < ActionController::TestCase
+
+  test "should import" do
+    post :import, { 'repo':'/does-not-exist' }
+    response = ActiveSupport::JSON.decode(@response.body)
+    assert_equal response['error']['message'], "Using file-based synchronization, but couldn't find /does-not-exist"
+    assert_response :error
+  end
```

Comment:
plain directory is fine, if you still have also "negative" path, feel free to keep it there too, both are testing different code paths